Meadowlark-js/backends/meadowlark-mongodb-backend/src/repository/ReferenceValidation.ts

```diff
@@ -5,22 +5,16 @@
 
```

Comment:
Mostly a rename of WriteHelper after moving out the one non-reference validation piece